app/assets/stylesheets/govuk_publishing_components/components/_emergency-banner.scss

```
@@ -0,0 +1,74 @@
+.gem-c-emergency-banner {
+  @include govuk-font(19);
+  background-color: govuk-colour("mid-grey");
+  color: govuk-colour("white");
+  margin-top: 2px;
+  padding: govuk-spacing(3) 0;
+
+  @include govuk-media-query($from: tablet) {
+    padding: govuk-spacing(6) 0;
+  }
+}
+
+.gem-c-emergency-banner--homepage {
+  border-bottom: govuk-spacing(1) solid govuk-colour("white");
+  border-top: govuk-spacing(1) solid govuk-colour("white");
+  margin-bottom: (0 - govuk-spacing(2));
```

This could be clearer if a [negative number is used][1]:

```suggestion
  margin-bottom: govuk-spacing(-2);
```

[1]: https://frontend.design-system.service.gov.uk/sass-api-reference/#example-using-negative-spacing